packages/ens-utils/src/registration.ts

```diff
@@ -22,3 +23,60 @@ export type Registration = {
   primaryStatus: PrimaryRegistrationStatus;
   secondaryStatus: SecondaryRegistrationStatus | null;
 };
+
+export const getDomainRegistration = (
+  expiryTimestamp: Timestamp,
+): Registration => {
+  const primaryStatus = getPrimaryRegistrationStatus(expiryTimestamp);
+  const secondaryStatus = getSecondaryRegistrationStatus(expiryTimestamp);
+  return {
+    expiryTimestamp,
+    primaryStatus,
+    secondaryStatus,
+    registrationTimestamp: null,
+    expirationTimestamp: expiryTimestamp,
```

Comment:
Let's remove this duplicated field from `Registration`.

Reply:
@lightwalker-eth yes! https://blockful.slack.com/archives/C03JQ5URTEG/p1720711089418559?thread_ts=1720654856.342799&cid=C03JQ5URTEG